# Computational methods in single-cell biology
### Project: "Benchmarking multimodal integration methods in single-cell genomics"
My code for the lecture _Computational Methods in single-cell biology_ in winter semester 2022/2023 at Helmholtz München (ICB) and Technische Universität München.

Note that the referenced data is not part of this repository, therefore my results are not reproducible just using the code from this repository.

The task of this project is to evaluate integration methods from the [scib](https://github.com/theislab/scib) library on data from the following publications:
* [Single-Cell RNA-Seq Reveals Cellular Heterogeneity of Pluripotency Transition and X Chromosome Dynamics during Early Mouse Development](https://doi.org/10.1016/j.celrep.2019.02.031)

* [The emergent landscape of the mouse gut endoderm at single-cell resolution](https://doi.org/10.1038/s41586-019-1127-1)

* [Characterisation of the transcriptional dynamics underpinning the function, fate, and migration of the mouse Anterior Visceral Endoderm](https://doi.org/10.1101/2021.06.25.449902)

* [Multi-omics profiling of mouse gastrulation at single-cell resolution](https://doi.org/10.1038/s41586-019-1825-8)

My analysis covers the method *scVI* (based on a conditional variational autoencoder) introduced in [Deep generative modeling for single-cell transcriptomics](https://doi.org/10.1038/s41592-018-0229-2).

